examples/postgres/Dockerfile.alpine.psql

```diff
@@ -1,10 +1,9 @@
-FROM liquibase/liquibase:latest-alpine
+FROM liquibase/liquibase:latest
```

Comment:
Here it goes `liquibase:alpine`  as per the docker team recommendation (https://hub.docker.com/layers/liquibase/liquibase/alpine/images/sha256-907389d258456030ecb50d8f077ee99025b7e71e0bed3efba7329a15a6fbfeda?context=explore)